TESTNET.md

```diff
@@ -0,0 +1,69 @@
+## TestNet User Guide
+
+A public TestNet is now online to help node operators practice running MobileCoin network infrastructure. Interested individuals and organizations are invited to participate and share feedback with the developers.
+
+In addition to bug reports and code contributions, we are seeking feedback on the clarity of our documentation, any criticisms you may have of our design, and any other constructive feedback you would like to share.
+
+MobileCoin Inc. will be announcing details on a formal bug bounty program in the coming weeks.
+
+### Sign up to receive TestNet *mobilecoins*
+
+Register to participate in the MobileCoin TestNet to receive an allocation of *mobilecoins*.
+
+1. Sign up using our [online form](https://forms.gle/ULNjA6cMxCD5XNyT7).
+
+1. You will receive an email containing a *master key* that you can use to claim *mobilecoins* in the TestNet ledger. Anyone who knows your *master key* will be able to spend your money, so keep it secret!
+
+### Send Your First Payment
+
+You can make your first MobileCoin payment right now!
+
+1. Download the TestNet client package for [mac](https://github.com/mobilecoinofficial/mobilecoin/releases/download/v0.1.4/MobileCoin.Testnet.dmg) or [linux](https://github.com/mobilecoinofficial/mobilecoin/releases/download/v0.1.4/mobilecoin-testnet-0.1.4-linux.tar.gz).
+
+1. Expand the package archive and launch the "MobileCoin TestNet" client, an app bundle (mac) or shell script (linux).
+
+1. Enter the *master key* you received in your email and confirm your balance.
+
+1. Try completing a payment to the sample request code you received in your email.
+
+1. Create your own payment request and share it in the [MobileCoin forum](https://community.mobilecoin.com).
+
+### Run a TestNet *Watcher Node*
+
+If you have a Linux-compatible home computer, or if you are willing to operate a Linux-compatible server in the cloud, you can run a *watcher node* in the MobileCoin TestNet.
```

Comment:
Should this say Docker-compatible or Linux-compatible?

Reply:
yeah, we'll add a comment to this effect later. thanks